app/admin/product.rb

```diff
@@ -0,0 +1,17 @@
+ActiveAdmin.register Product do
+
+# See permitted parameters documentation:
+# https://github.com/activeadmin/activeadmin/blob/master/docs/2-resource-customization.md#setting-up-strong-parameters
+#
+# permit_params :list, :of, :attributes, :on, :model
```

Comment:
Need the products to be editable. Need to configure this file. Show sales of each products too and the shop they belong to.